RELEASE.md

```diff
@@ -124,69 +123,22 @@ Remove all `Thanks @user!` references unless it is from a community member, note
 
 Go through PR descriptions and determine what should be called out in the release, what should be summarised, or left out completely.
 
-Copy all unique PR descriptions into template under the `## What's New` heading.
-
-Group items and add headings to call out sections to have the release start to take shape, add in `>` quotes as editors notes for ideas on topics.
-
-Publish a draft when there is an overall story and get others to review.
-
-Finalise content based on feedback, clean up text, add emojis.
-
-Copy original description from `Version Packages` to the `View verbose release notes` section.
-
-Publish release on GitHub.
-
-## Website release
-
-Add a new release page to website for this release under `/docs/pages/releases/YYYY-MM-DD.mdx` using the same format as previous release.
-
-Copy over the GitHub markdown content from the GitHub release into this document (excluding verbose notes).
-
-Add release to release page index under `/docs/pages/releases/index.mdx` with a summarised heading.
-
-Commit website update and open PR to add to `main`.
-
-Have PR reviewed and merged into `main`.
-
 ## GitHub branch sync
 
 When we do a release we need to make sure `main` and `website_live` are both in sync with each other.
 
-Before doing this, ensure nothing is merged into `main` or `website_live` for the time it takes to complete the following (around 15 minutes).
-
-Create a branch off `main` such as `bring-in-latest-website-changes` and merge in `website_live` changes, open a PR and get it merged, example - <https://github.com/keystonejs/keystone/pull/6470>
-
-Then the other way around, create a branch off `website_live` such as `bring-in-latest-main-changes`, and merge in `main` changes, open a PR and get it merged, example - <https://github.com/keystonejs/keystone/pull/6472>
+This can be done by checking out `website_live`.
 
-Once this is done, the histories will be out of sync, GitHub will state that `website_live` is still x commits behind and x commits ahead, if you look at the branches page - <https://github.com/keystonejs/keystone/branches>
+Then running:
 
-To resolve this in the CLI (first on the `website_live` side):
-
-Go to the `website_live` branch
-`git checkout website_live`
-
-Check that we're identical to main
-`git diff main`
-
-Do a merge that should just update the parents of the new commit
-`git merge main`
-
-This should be empty
-`git diff origin/website_live`
-
-This should be empty
-`git diff main`
-
-Force push `website_live` (after turning off branch protection in github)
-`git push --force`
+```sh
+EDITOR=subl git rebase --interactive origin/main
```

Comment:
```suggestion
git rebase --interactive origin/main
```

[Setting up your preferred editor for `git` is on the first page of the `git` setup guide.](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup)

I think we can omit that.